Doc/Doc.pm

```diff
@@ -683,12 +696,17 @@ sub scan {
   my $phash = $parser->{SYMHASH};
   my $n = 0;
   while (my ($key,$val) = each %$phash) {
-    #print "adding '$key'\n";
+
     $n++;
 
     $val->{File} = $file2;
-    $hash->{$key} = $val
+    #set up the 3-layer hash/database structure: $hash->{funcname}->{PDL::SomeModule} = $val
+    if (defined($val->{Module})){
+	$hash->{$key}->{$val->{Module}} = $val;
+    } else {
+	print STDERR "no Module for $key in $file2\n";
```

Comment:
Not a `warn`?

Reply:
good catch.  mostly that was for debugging during development but it's good to keep in there.